desc/coils.py

```diff
@@ -267,6 +269,8 @@ def to_SplineXYZ(self, knots=None, grid=None, method="cubic", name=""):
             - `'catmull-rom'`: C1 cubic centripetal "tension" splines
         name : str
             name for this coil
+        discontinuous_indices : ndarray
+            indices of knots at which the curve is discontinuous (i.e. only C0)
```

Comment:
Maybe there is a better name for discontinuous indices, something more like "section indices" or something idk. Or you can keep this name. but you should explain more in the docstring what it actually means for the spline, i.e. the spline will not be evaluated as a periodic spline curve with the given knots, but instead as a sequence of splines over the knots with start and endpoint given by discnontinuous_indices[i-1,i] for the ith spline (with 0: disc_index[0] as the first interval and knots[disc_index[-1]], knots[01] as the last interval) evaluated with whatever endpoint conditions it is, (maybe say default endpoint conditions and refer to interpax's docs), and at knots[discontinuous_indices], the spline

Reply:
I agree, I think I was confused about what this PR is doing because of the misleading names. This isn't really a discontinuous C0 spline, it's a regular spline of some order >0 that is only C0 in some intervals. And the whole curve is still continuous (at least C0 everywhere). Right?

Reply:
You're right Daniel and it is confusing so I'll fix it. If you consider the whole coil then it is C0 continuous. The "discontinuous" refers to the fact that it's discontinuous in it's derivatives at those indices. And in between those indices, it is a spline whose interpolation is chosen by the user

Reply:
I think I will use `break_indices`. And this is a possible docstring:

"Indices of knots at which the curve breaks and is only continuous in space (and not continuous in it's derivatives). In between each set of break points, there is an unbroken spline whose start and endpoints are given by `break_indices[i-1, i]`, where `i` indicates the ith spline. Each (the ith) spline is interpolated independently of all other unbroken splines (i+1th, i-1th, etc.) and does not consider their query points (knots) when interpolating. The boundary conditions are evaluated using interpax's default () where non-periodicity is assumed."

Reply:
I'd say 
```
and is only C0 continuous (eg, continuous but with "corners" where the derivative jumps)
```